lib/app.js

```diff
@@ -173,9 +173,13 @@ if (navigator.mozSetMessageHandler) {
   navigator.mozSetMessageHandler('activity', function(activityRequest) {
```

Comment:
This isn't portable, but it doesn't look like WebKit or Blink have an equivalent function. Maybe fall back to ```navigator.setMessageHandler()```?